HPXMLtoOpenStudio/resources/meta_measure.rb

```diff
@@ -312,29 +312,24 @@ def get_measure_instance(measure_rb_path)
 
 # TODO
 #
-# @param measure_args [TODO] TODO
-# @param provided_args [TODO] TODO
-# @param lookup_file [TODO] TODO
-# @param measure_name [TODO] TODO
+# @param measure_args [OpenStudio::Measure::OSArgumentVector] Array of OpenStudio Measure Arguments
+# @param provided_args [Hash] Map of provided measure arguments to values
+# @param measure_name [String] OpenStudio-HPXML measure directory name
 # @param runner [OpenStudio::Measure::OSRunner] Object typically used to display warnings
-# @return [TODO] TODO
-def validate_measure_args(measure_args, provided_args, lookup_file, measure_name, runner = nil)
+# @return [Hash] Map of provided measure arguments to values
+def validate_measure_args(measure_args, provided_args, measure_name, runner = nil)
```

Comment:
I'm seeing that ResStock calls this method and passes `lookup_file`.

Reply:
Hmm, from where?

Reply:
Oh, nvm. I see it.